css/.scss-lint.yml

```diff
@@ -210,6 +214,11 @@ linters:
     SpaceAfterPropertyName:
         enabled: true
 
+    # Checks for spaces following the name of a variable and before the colon
+    # separating the variables's name from its value.
```

Comment:
I thought it was saying that it will check to *ensure* there is a space before a colon in variable declarations, so `$foo : bar;`. It’s not (it checks for *no* space, as we want). But comment should be clarified.